### ISSUE TITLE
Unhandled Runtime Error

### ISSUE THREAD

**Issue by Suryanshu1005** [WebDevSimplified/next-13-todo-list#1]
I don't know what might be causing the error but I think it is the missing .env file. 

![image](https://github.com/WebDevSimplified/next-13-todo-list/assets/59340699/ef0399ba-32f4-438d-b5c6-6115c58ec983)

**Comment by eno-conan** [comment]
@Suryanshu1005 
I did following procedure. Not cause the above error.
I would be happy to help you!

1. Run `git clone https://github.com/WebDevSimplified/next-13-todo-list.git`
2. Run `npm i`
3. Create file `.env` to project root directory
4. Write "Connect information" to `.env`
(I referred to [this repository](https://github.com/remix-run/indie-stack) for reference.)
    ```
    DATABASE_URL="file:./data.db?connection_limit=1"
    ```
5. Run `npx prisma migrate dev --name init`
  - terminal output
    <img width="492" alt="image" src="https://github.com/WebDevSimplified/next-13-todo-list/assets/62947052/b8135ac5-b2c7-4ac4-a884-c3191f4e3660">
6. Run `npm run dev`
